src/components/Reviews/Reviews.vue

```diff
@@ -35,22 +35,25 @@ import axios from 'axios';
 
             averageRating(){
                 console.log(this.reviews)
-                this.reviews.array.forEach(review => {
-                    this.averageRating += review.starRating
-                });
-                this.averageRating = this.averageRating / this.reviews.length;
+                for(let i = 0; i < this.reviews.length; i++){
+                    this.averageReview = this.averageReview + this.reviews[i].starValue;
+                }
+                this.averageReview = (this.averageReview / this.reviews.length).toFixed(1);
```

Comment:
tofixed(2) would be consistent with my review preview. Looking at airbnb they have some weird conditioning if its 5.0 it will be 1 decimal, but otherwise will do 2 decimal places.